test/transmuxer/transmuxer_integration.js

```diff
@@ -293,6 +293,33 @@ describe('Transmuxer Player', () => {
       await player.unload();
     });
 
+    it('H.265+AAC in TS', async () => {
+      const chromeVersion = shaka.util.Platform.chromeVersion();
+      if (shaka.util.Platform.isWindows() &&
+          chromeVersion && chromeVersion === 117) {
+        // It appears that Chrome 117 beta in Windows is incorrectly reporting
+        // H.265 in MediaCapabilities
```

Comment:
Does this also currently affect dev and canary builds from 118?  You might want to do chromeVersion >= 117 until this is confirmed fixed.

Also, please put a link to the bug like http://crbug.com/BUG_NUMBER_HERE so that we can follow-up and remove the exclusion when it is fixed.

Reply:
Only dev 118